.circleci/config.yml

```diff
@@ -77,10 +35,84 @@ jobs:
           command: |
             cd drupal/web && php core/scripts/run-tests.sh --verbose --sqlite --module elasticsearch_helper
 
+jobs:
+  build:
+    docker:
+      - image: wunderio/circleci-codeception-base
+
+    steps:
+      - run:
+          name: Create Drupal
+          command: |
+            composer create-project drupal/recommended-project:8.8.6 drupal
+            cd drupal
+            COMPOSER_MEMORY_LIMIT=-1 composer require drupal/core-dev:^8
+            COMPOSER_MEMORY_LIMIT=-1 composer require drush/drush:9
+            COMPOSER_MEMORY_LIMIT=-1 composer require elasticsearch/elasticsearch:~6.0
+      - save_cache:
+          paths:
+            - ~/project/drupal
+          key: v2-composer-dependencies-{{ .Environment.CIRCLE_SHA1 }}
+
+  test-es-6:
+    docker:
+      - image: wunderio/circleci-codeception-base
+      - image: circleci/mariadb:10.2
+        environment:
+          MYSQL_DATABASE: drupal
+          MYSQL_ROOT_PASSWORD: dbpassword
+          MYSQL_ROOT_HOST: "%"
+      - image: docker.elastic.co/elasticsearch/elasticsearch:6.8.3
+        environment:
+          - discovery.type=single-node
+          - http.host=0.0.0.0
+          - transport.host=127.0.0.1
+          - xpack.security.enabled=false
+          - cluster.name=docker-cluster
+          - bootstrap.memory_lock=true
+          - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+
+    steps:
+      - checkout:
+          path: ~/project/drupal/web/modules/elasticsearch_helper
+      - restore_cache:
+          keys:
+            - v2-composer-dependencies-{{ .Environment.CIRCLE_SHA1 }}
+      - start-drupal-test
```

Comment:
Is there a way to reuse this block and change the Elasticsearch version via a parameter that is passed from the workflow jobs? 

- image: docker.elastic.co/elasticsearch/elasticsearch:<< version >>

Reply:
Apparently not. I tried re-usable jobs yesterday but the parameter scope does not seem to reach the docker part